src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -308,9 +315,14 @@ def acr_build(cmd,
             is_push_enabled = False
             logger.warning("'--image -t' is not provided. Skipping image push after build.")
 
+    if os_type:
+        platform = PlatformProperties(os_type=os_type)
+    else:
+        platform = PlatformProperties(os_type='Linux')
+
     build_request = QuickBuildRequest(
         source_location=source_location,
-        platform=PlatformProperties(os_type='Linux'),
+        platform=platform,
```

Comment:
You can keep it as `PlatformProperties(os_type=os_type)` but default `os_type='Linux'` in the parameter.

The above if-else block is redundant.